### DISCUSSION TITLE
Inserting images inline

### DISCUSSION THREAD

**Discussion by Pantomation** [atom/atom#23348] [Q&A]
I would like to create a package that replaces an image path in python code with the image inline.
Kind of like this:
![image](https://user-images.githubusercontent.com/61744678/145599729-af1dc182-867a-467a-85d8-0743a492f6f0.png)

However, reading through the API: https://flight-manual.atom.io/api/v1.57.0/AtomEnvironment/
It seems one can only insert text.

How can I insert an `<img />` tag everywhere there is a string that matches eg. *.png?

Is it possible?

**Comment by Pantomation** [comment]
So I managed to insert an image tag via the inspector at the desired location, but how can I modify the rendered html through a package?

**Reply by ThatXliner** [reply]
You can just modify the DOM
